packages/sdk/src/mpl-token-authorization-rules.ts

```diff
@@ -1,5 +0,0 @@
-export * from './errors';
```

Comment:
This file was just added a month ago, I think it was needed to export everything properly.  Instead of deleting it, shouldn't you be adding?:
```
export * from ./pda;
```

Reply:
This is actually the old ts file from when it was called mpl-token-authorization-rules. There's another file mpl-token-auth-rules.ts

Reply:
ah I see